LICENSE

```diff
@@ -0,0 +1,40 @@
+SPRYKER SYSTEMS GMBH EVALUATION LICENSE AGREEMENT
```

Comment:
has to be removed or clarified with legal.

Reply:
Why? Isn't it the same as for any other module?

Reply:
no, it's not part of Spryker SCOS

Reply:
But it's pretty important nowadays.

Reply:
confirmed with Anneliese Hartlaub that we need to use this license.